### PR TITLE
Making forcedroid work for native app generation

### PR DESCRIPTION
**Pull request by bhariharan**
Outstanding Items:
* Script to bring in the ```Gradle``` wrapper.
* Make it all work while importing into ```Android Studio```.

### CODE REVIEW

File: node/forcedroid.js [hunk 1]
```diff
@@ -392,6 +394,50 @@ function createAppGradleFile(appFolderName, appName, usesSmartStore) {
 }
 
 //
+// Fixes library dependency references in the app's build.gradle files
+//
+function fixAppGradleFiles(appFolderName, appName, usesSmartStore) {
+    console.log('Tweaking build.gradle in ' + appFolderName + "/" + appName);
```
Comment (bhariharan) [inline node/forcedroid.js]: All the ```Gradle``` dependencies are path based at this point, so tweaking is required post generation :-(

**Comment by wmathurin** [comment]
Looks reasonable

**Comment by bhariharan** [comment]
Brought in the ```Gradle``` wrapper [here](https://github.com/forcedotcom/SalesforceMobileSDK-Android/commit/7d3b2e7785d4dc47eb38a854d5bb31f407035961). Only outstanding item is some ```groupId``` conflict - needs tweaking of a line, but otherwise ```Android Studio``` import works. Onto ```hybrid``` now.

**Comment by wmathurin** [comment]
:+1:

**Pull request merged**